pythonAPI/germanetpy_tutorial.ipynb

```diff
@@ -463,37 +543,97 @@
     "pycharm": {}
    },
    "source": [
-    "## Example for IC-based measures\n",
+    "### Example for IC-based measures\n",
     "\n",
-    "Information-content-based measures augment the structural distances, captured by the taxonomy with word frequencies. Thus the problem of lacking uniform distances in the Graph can be reduced by providing additional information about the typicality of words. The frequencies are used to compute the information content, which graduates concepts from specific to general. If a very specific synset is compared to a very general one, the relatedness will be low. The relatedness is measured based on the information content of the lowest common subsumer (the lowest synset in the hierachy that is hypernym to both synsets that are compared to each other).\n",
+    "One problem with path-based measures is that paths of the same length in the hypernym relation can correspond to very different intuitive semantic \"distances\".\n",
+    "Measures based on *information content* (IC) seek to solve this problem by augmenting information about the structural distances in the hypernym relation with information about word frequencies. \n",
+    "The word frequencies are used to compute the information content, which grades concepts from more specific to more general. If a very specific synset is compared to a very general one, the relatedness will be low. The relatedness of two synsets measured based on the information content of their least common subsumer (the lowest synset in the hierarchy that is hypernym to both synsets).\n",
```

Comment:
noch ein typo von mir:
The relatedness of two synsets measured based on the information content --> da fehlt ein "is", können wir das da noch reinschreiben?

Reply:
Klar, korrigiert in 88e698d